core/src/main/java/org/elasticsearch/action/bulk/BulkProcessor.java

```diff
@@ -161,6 +163,14 @@ public Builder setThreadPool(ThreadPool threadPool) {
             return this;
         }
 
+        /**
+         * Sets an optional BiFunction to schedule flush actions and request retries
+         */
+        public Builder setScheduleFunction(BiFunction<TimeValue, Runnable, ScheduledFuture<?>> scheduleFunction) {
+            this.scheduleFunction = scheduleFunction;
```

Comment:
I don't see this member being used anywhere? Maybe I am missing something? I also wonder if we should have a test setting this as well so we can check that it's actaully used?